helsingborg-stad/aterbruket-webapp
# Fix/mini header when scroll

**Explain the changes you’ve made**
I've changed so the header gets smaller when starting to scroll
 
**Explain why these changes are made**
Because the header took a lot of space on the screen, better if it is smaller so you see more of the content
 
**Explain your solution**
Added scroll event to the header so it reacts to scroll, and then makes the h2-tag inside the header scale to smaller and made it center, and changing the height on the header.
**_ALSO_**: because the "scan-button" is in another components I used vanilla JS to find the element by ID and make it change position when scrolled too. (had problem with doing it the react-way, so made it vanilla style)
 
**How to test the changes?**
Scroll up and down in all the views and make sure that it looks nice
**_OBS_**! it looks way better if you test it in this link (https://testanna.d21bj89y0jdxx1.amplifyapp.com/) and download it to your phone, rather then localhost (pga stupid download-btn :) ).
